app/src/main/java/com/zeapo/pwdstore/SearchableRepositoryViewModel.kt

```diff
@@ -266,7 +266,7 @@ class SearchableRepositoryViewModel(application: Application) : AndroidViewModel
 
     private fun listFilesRecursively(dir: File): Flow<File> {
         return dir
-            .walkTopDown().onEnter { file -> shouldTake(file) }
+            .walkTopDown().onEnter { file -> if (file == dir) true else shouldTake(file) }
```

Comment:
```suggestion
            // Take top directory even if it is hidden.
            .walkTopDown().onEnter { file -> file == dir || shouldTake(file) }
```

Reply:
@Hazegard I find it a bit clearer this way, what do you think?

Reply:
Thank you for your suggestion, I agree it's better!